cmake/toolchains/riscv32.cmake

```diff
@@ -0,0 +1,19 @@
+# evmone: Ethereum Virtual Machine
+# Copyright 2023 Pawel Bylica.
+# Licensed under the Apache License, Version 2.0. See the LICENSE file.
+
+set(RISCV /usr/local/riscv)
+
+set(CMAKE_SYSTEM_PROCESSOR riscv32)
```

Comment:
Can we specify the riscv32imac triple?

Reply:
This value is only informational, but you can modify the compiler flags.

By default it builds as `rv32imafdc`. If we switch to `rv32imac` it will not build because the included glibc is not compatible (the ABI for floating-point values is different). We can configure and build the toolchain ourselves but seems to be a lot of work for the start.